doc/mp-build.md

```diff
@@ -86,3 +86,10 @@ A pak file is a zip file created using the deflate compression algorithm. The co
 
 ### Logs
 Logs from build process are written to the `logs/build.log` file. This is useful for debugging purposes in case the build fails.
+
+### Next steps
```

Comment:
This was part of a 'Troubleshooting' guide before. If we're removing that in favor of the one in the main README, "Next steps" doesn't make sense here as a title. I'd suggest just removing this section altogether. (same for the other pages)